.babelrc

```diff
@@ -1,3 +1,8 @@
 {
-  "presets": ["es2015", "stage-0", "react"]
+  "presets": ["es2015", "stage-0", "react"],
+  "env": {
+    "development": {
+      "presets": ["react-hmre"]
```

Comment:
Yep, I'm aware of it. According to Dan Abramov a new version is on the horizon, but no news yet. If it starts causing problems we can just remove it and fall back to only using browserify-hmr (that'll give the same behaviour as we have today, without this change).